### PR TITLE
Change gets to posts

### PR DESCRIPTION
**Pull request by davidskalinder**
...In everything except the article-level functionality.

Issue thread is at https://github.com/davidskalinder/mpeds-coder/issues/101

I think this one should be user-invisible and has been running in our production deployment for a couple weeks now, so it should be safe to accept unless I'm missing something subtle...

### CODE REVIEW

**Comment by davidskalinder** [comment]
Also, to be technically precise: after these changes there should still be a few idempotent `GET`s left in the code, but all the ones that change the state of the world should be changed to `POST`s.

**Review by alexhanna — APPROVED** [review]

File: mpeds_coder.py [hunk 1]
```diff
@@ -1351,10 +1351,10 @@ def changeCode(pn):
     return jsonify(result={"status": 200})
 
 
-@app.route('/_mark_ec_done')
+@app.route('/_mark_ec_done', methods=['POST'])
```
Comment (davidskalinder) [inline mpeds_coder.py:1354]: Hmm, it looks like this change got clobbered by a subsequent merge somehow.  I'm guessing it'll be easier to just add it back unless I find that it's a symptom of some larger disease?  @alexhanna, do you want to just change it on your side?

**Comment by alexhanna** [comment]
Ack, crap. Yeah, probably just easier to add this back in my fork and send up a PR.

**Comment by davidskalinder** [comment]
> Ack, crap. Yeah, probably just easier to add this back in my fork and send up a PR.

I blame past-me again.  I'll let you know if I find others, but so far the diff between our master and the new upstream is looking pretty clean...

**Comment by alexhanna** [comment]
Thanks for finding it, regardless.

**Pull request merged**